modules/documents/client/views/document-manager.html

```diff
@@ -507,7 +507,13 @@ <h2 title="{{documentMgr.infoPanel.data.displayName | removeExtension}}">{{docum
 									<span class="name">ID:</span>
 									<span class="value">{{ documentMgr.infoPanel.data.documentId }}</span>
 								</li>
-								<li ng-if="documentMgr.infoPanel.data.keywords.length > 0">
+								<!--
```

Comment:
One thought is to create a filter, passing in some config params, and then pass in the whole object instead of the property so that you can manage all the special casing inside the filter which returns you the desired output (in this case, text).